server/views/layout.njk

```diff
@@ -121,14 +121,14 @@
 
 {% block footer %}
   {%- from "govuk/components/footer/macro.njk" import govukFooter -%}
+  {% set footerItems = [{ href: "/cookies-policy", text: "Cookies" }] %}
```

Comment:
The content of the privacy policy is very very Send Legal Mail centric, so it did not feel right that it appears in the footer for the mail room journey (though there is no specific AC about that in the ticket itself)

Reply:
Yes there is no privacy policy on DPS